tests/testing_utils.h

```diff
@@ -127,25 +131,20 @@ Catch::Generators::GeneratorWrapper<std::tuple<T1, T2, T3>> xsf_test_cases(
     );
 }
 
-
 template <typename T>
 T adjust_tolerance(T tol) {
     // Add some wiggle room to tolerance from table.
     return 4 * std::max(std::numeric_limits<T>::epsilon(), tol);
```

Comment:
For failures due to different compiler versions, like the currently failing Linux tests, I think it's fine just to increase the wiggle room factor here for now, as long as it doesn't get too large.